tests/Microsoft.Identity.Test.Unit/BrokerTests/RuntimeBrokerTests.cs

```diff
@@ -53,6 +53,7 @@ public void Init()
             _coreUIParent = new CoreUIParent() { SynchronizationContext = _synchronizationContext };
             ApplicationConfiguration applicationConfiguration = new ApplicationConfiguration(isConfidentialClient: false);
             _logger = Substitute.For<ILoggerAdapter>();
+            _logger.PiiLoggingEnabled.Returns(true);
```

Comment:
This doesn't do anything, since there are not pii related tests in this class? Are they missing?

Reply:
Added WamUsernamePasswordRequestAsync_WithPiiAsync